keyboards/ergo42/rev1/config.h

```diff
@@ -67,9 +67,11 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 /* ws2812 RGB LED */
 #define RGB_DI_PIN D3
 #define RGBLIGHT_TIMER
-#define RGBLED_NUM 12    // Number of LEDs
+#define RGBLED_NUM 14    // Number of LEDs
```

Comment:
This should be redefined in your keymap.  You can "`#undef`" it first, and then define it.